src/theory/quantifiers/trigger.h

```diff
@@ -33,102 +32,180 @@ namespace inst {
 
 class IMGenerator;
 class InstMatchGenerator;
-}/* CVC4::theory::inst namespace */
-}/* CVC4::theory namespace */
-}/* CVC4 namespace */
-
-
-namespace CVC4 {
-namespace theory {
-namespace inst {
 
+/** Information about a node used in a trigger. */
```

Comment:
This class is a bit confusing.  To improve, I suggest:
1. Move the details of what the class is doing to the class comment: explain what is stored and how it should be used.
2. Move the data to a private section of the class and add methods to manipulate the data
3. Give the data slightly longer, more understandable names.

Reply:
I suggest doing 2 and 3 on a separate commit to keep the changes on this PR small.  What do you think?

I'll make another documentation-only pass today and fix the typos you point out.

Reply:
Btw, in retrospect, I would have done these steps for this commit:

- Before making any changes to Trigger/InstMatchGenerator, I would have made a separate PR that refactors and documents Trigger/InstMatchGenerator,
- After that is approved and merged, I would have made the small change to Trigger/InstMatchGenerator required by this PR.

Notice that these changes were made to Trigger/InstMatchGenerator in July before we instituted the new policies, so I did not know to plan in this way.

Reply:
Re: making other changes on a separate PR, sounds good.

Re: in retrospect...yes, I think that would have made it easier, but in any case we will press forward with this one as is to get it through.